.github/workflows/stale.yml

```diff
@@ -27,8 +27,7 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         days-before-stale: 180
         days-before-close: 14
-        stale-pr-label: 'stale'
+        exempt-issue-labels: 'discussion-url'
```

Comment:
Any better suggestion for the label?

Reply:
`discussions-to` maybe (so it matches the header field name in the EIP)?

Reply:
I'm fine with that, though the value is an uri. Perhaps we should rename it to `discussion-uri` both label and in the EIPs? :)

Reply:
I would approve a PR that changes it to `discussion-uri` everywhere.  If you are going to immediately follow this PR up with all of the necessary PRs to change to `discussion-uri` I'm fine with leaving this as is.  If that is just an idea to put on the backburner, I think we should maintain consistency for now and have the label match the field.

Reply:
Let me know which way you would like to go, or if you would like to continue discussion, so I can know to merge or not.

Reply:
Renamed to `discussions-to` here just to get this merged. We can make a bigger PR later and renaming the label is once off on Github, all tagged issues carry it over.